src/content/docs/companion/optimization.md

```
@@ -20,7 +17,7 @@ You can adjust this setting if needed:
 - Decrease limit to only use closer, potentially more reliable fixes
 
 To modify the maximum distance, send an MQTT message:
-```markdown
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)